docs/docs/07-forms.md

```diff
@@ -166,3 +166,44 @@ To make an uncontrolled component, `defaultValue` is used instead.
 > Note:
 >
 > You can pass an array into the `value` attribute, allowing you to select multiple options in a `select` tag: `<select multiple={true} value={['B', 'C']}>`.
+
+### Submit form
+
+If you need to programmatically submit a form, you have to obtain a [reference to the DOM node](08.1-more-about-refs.md), and manually call `submit()` on it.
+
+```javascript
+var MyComponent = React.createClass({
+  handleClick: function() {
+    if (this.myForm !== null) {
+      this.myForm.submit();
+    }
+  },
+  render: function() {
+    return (
+      <div>
+        <form href="http://example.com/location" method="post" ref={(ref) => this.myForm = ref}>
```

Comment:
I would break earlier so that `ref` is more visible. Right now the important part is easy to miss. Maybe do it this way:

```js
<form
  href="..."
  method=".."
  ref={...}>
  ...
</form>
```
?